test/integration/integration_suite_test.go

```diff
@@ -212,19 +212,30 @@ var _ = AfterSuite(func(done Done) {
 		close(done)
 		return
 	}
+
+	if kubeCtl == nil {
+		logf.Log.Info("No cluster found")
```

Comment:
why is this info message. isnt this an error.

Reply:
No this means kubectl is not configured, means setup is not complete, some error happened during BeforeSuite.